tests/functional/u/useless/useless_return.py

```diff
@@ -13,3 +13,31 @@ def mymethod(self): # [useless-return]
     # These are not emitted
     def item_at(self):
         return None
+
+
+def function2(parameter):  # [useless-return]
+    if parameter:
+        pass
+    return
+
+
+def function3(parameter):  # [useless-return]
+    if parameter:
+        pass
+    else:
+        return
+
+
+def function4(parameter):  # [useless-return]
+    try:
+        parameter.do()
+    except RuntimeError:
+        parameter.other()
+        return
+
+
+def function5(parameter):  # [useless-return]
```

Comment:
I'd like to see some cases where there is actual code after these `excepts` and `if ... else` statements to make sure we are not raising false positives.